css-layout-api/Overview.bs

```diff
@@ -1969,20 +1975,20 @@ following steps:
 
             1. Let |fragmentResultValue| be |value|.
 
-    16. If |fragmentResultValue| is a [=platform object=]:
+    16. If |fragmentResultValue| [=is a platform object=]:
```

Comment:
This is a very strange way of doing it. I would expect `If |fragmentResultValue| [=implements=] {{FragmentResult}}`.

What the current spec text is doing means that

```js
const possibleValue = new Blob();
possibleValue.inlineSize = 5;
```

cannot be used. Whereas every other web API that accepts a dictionary, using normal Web IDL semantics, would allow it.

The root problem here appears to be overuse of the `Function` IDL callback type instead of declaring a callback that returns a proper Web IDL type, in this case `(FragmentResult or FragmentResultOptions)`.

I don't know what the current implementation is, and so I don't know if we can fix this. But I don't think we should export the internal "is a platform object" for use by external specs because of misuse like this.

Reply:
I can undo this line of change (though I think this text is trying to invoke that algorithm, even if we'd like it not to), if we prefer that to acknowledging that it's relying on a non-exported definition.

What I would prefer not to do is block this PR on a deep dive of CSS Layout API's choice here.

Reply:
Yeah, we should not be discussing API design in this PR. If there's an issue, it should be raised separately; this is just a syntax transform.

Reply:
My preference would be to merge this PR as-is and open a new issue. We should not block on deleting https://github.com/w3c/css-houdini-drafts/pull/1127#discussion_r1728015837

Reply:
Deleting the manual anchor was just a "proper use of Bikeshed" thing; whether or not the anchor is exported was irrelevant, and just means it needed a link-defaults line instead to allow it grab a non-exported definition. I've fixed it.